### PR TITLE
Fix service syntax and broken links

### PR DESCRIPTION
**Pull request by xpicox**
Fixes to the `service` syntax:
- Postfix type specification in parameter declaration
- Added access modifiers

### CODE REVIEW

File: web/language-tools-and-standard-library/basics/modules.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ Jolie also supports the importing of all the (public) symbols defined in a given
 
 The `service` is a component that includes blocks that define its deployment and its behaviour. 
 
-More precisely, a service node contains a collection of Jolie components, like named procedures and communication ports, and may specify a typed value used to parametrise its execution &mdash; this value can be passed either from the execution command used to launch the Jolie interpreter as well as by an importer that wants to use the service internally (e.g., see [embedding](../language-tools-and-standard-library/architectural-composition/embedding). The syntax of `service` definition is the following:
+More precisely, a service node contains a collection of Jolie components, like named procedures and communication ports, and may specify a typed value used to parametrise its execution &mdash; this value can be passed either from the execution command used to launch the Jolie interpreter as well as by an importer that wants to use the service internally (e.g., see [embedding](../architectural-composition/embedding.md)). The syntax of `service` definition is the following:
```
Comment (thesave) [inline web/language-tools-and-standard-library/basics/modules.md]: The link should not end with the source extension (so `embedding` and not `embedding.md`)

**Comment by thesave** [comment]
Thanks for the PR!
It seems fine, I just added a small comment for a fix :)

File: web/language-tools-and-standard-library/basics/modules.md (hunk 1 above)
Comment (xpicox) [reply web/language-tools-and-standard-library/basics/modules.md]: I have built the docs locally before making the PR and I had to add the source extension in order to get the link working. Anyway I will remove the extension to see if its a problem affecting only the docker image in `v1.10.x` or if the links started to behave differently with the new docs 😄

**Comment by thesave** [comment]
Thanks!

**Pull request merged**